torch/csrc/api/src/nn/modules/activation.cpp

```diff
@@ -96,5 +96,32 @@ void LogSigmoidImpl::pretty_print(std::ostream& stream) const {
   stream << "torch::nn::LogSigmoid()";
 }
 
+// ============================================================================
+
+SoftmaxImpl::SoftmaxImpl(const SoftmaxOptions& options_)
+    : options(options_) {}
+
+void SoftmaxImpl::reset() {}
+
+void SoftmaxImpl::pretty_print(std::ostream& stream) const {
+  int dim = options.dim();
+  torch::Dtype dtype = options.dtype();
+  stream << "torch::nn::Softmax(";
+  if (dim != -1) {
+    stream << "dim=" << options.dim();
+    if (dtype != torch::Dtype::Undefined) {
+      stream << ", ";
+    }
+  }
+  if (dtype != torch::Dtype::Undefined) {
+    stream << "dtype=" << options.dtype();
+  }
+  stream << ")";
```

Comment:
To match the Python version, we can just print `dim` in pretty_print.

Reply:
Fixed it to print `dim` only.